server/db/schemas/test_results.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE test_results(
+    test_id SERIAL PRIMARY KEY,
```

Comment:
Nitpick: For consistency all the other schema define the primary key with the foreign key constraints at the bottom but this doesn't actually change anything so up to you.

```suggestion
    test_id SERIAL,
```